CodeForAfrica/DebunkBot
# Read from twitter stream

## Description
App can read data from twitter stream by passing in trigger keywords

Finishes #172329985

## Type of change

Please delete options that are not relevant.

## How to test
> - set up env keys using `.env.sample` as a guide
> - start the app: `docker-compose up`
> - visit `http://localhost:8000/stream/` in your browser
> - create a tweet which includes the word `asciidev`
> - check your terminal for as the tweet streams there

## Checklist:
